.github/workflows/test-al2.yml

```diff
@@ -0,0 +1,22 @@
+name: AL2 package build
+
+on:
+  push:
+    branches: [ "1.x" ]
+  pull_request:
+    branches: [ "1.x" ]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
Just want to make sure I'm following this right. Please forgive my ignorance. 

We're running an AL2 container on an ubuntu instance for the test?

Reply:
Yeah, it's the simplest way to get an AL2 environment in GitHub Actions. Otherwise we'd need to construct a more complicated customer runner setup with instances running in some AWS account that we'd need to own.